.github/workflows/push_image.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           registry: quay.io
       - name: get short sha
         run: echo "short_sha=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+      - name: build and push bytecode image
+        run: make build-and-push-bc-image
```

Comment:
Yo can push to the `workflow-test` branch to test this - I just did it, it triggers this job: https://github.com/netobserv/netobserv-ebpf-agent/actions/runs/12254212563

Reply:
yeah that looks good
https://quay.io/repository/netobserv/ebpf-bytecode/manifest/sha256:613c42652550bd164a4582678ff06dcff54154b230fc22391c72eb72440a8664